notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>Also suggest using pathlib here, is this going to be using Zenodo or Google Cloud long term?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Yes. I intend to replace all the harvest paths with gcloud paths to the wanted files. </p>